src/main/java/mate/academy/Main.java

```diff
@@ -1,11 +1,13 @@
 package mate.academy;
 
+import mate.academy.lib.Injector;
 import mate.academy.model.Movie;
 import mate.academy.service.MovieService;
 
 public class Main {
     public static void main(String[] args) {
-        MovieService movieService = null;
+        Injector injector = Injector.getInstance("mate.academy");
+        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
```

Comment:
Implement task : 
in the mate/academy/Main.main() method create instances of new services using injector and test all their methods

Reply:
not fixed